apps/pie-storybook/turbo.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "extends": [
+    "//"
+  ],
+  "pipeline": {
+    "dev": {
```

Comment:
`pie-storybook` also has a `build` script. Should it be included here?

Reply:
Good question. I just tried this and it runs just fine without it (because its definition would be identical to the one in the root `turbo.json`). I think this means that I should be able to remove the one that I've defined in `pie-docs/turbo.json` though!